packages/ssh-tunnel/src/index.ts

```diff
@@ -187,6 +136,7 @@ export class SshTunnel extends EventEmitter {
       // close error
       this.serverClose().catch<Error>((e) => e),
       this.closeSshClient(),
+      // TODO: shouldn't we close the open connections first since they depend on the others?
       this.closeOpenConnections(),
```

Comment:
good question, there is the `.catch<Error>((e) => e)` with the comment "we want to await on closing the connections before re-throwing server", that would probably make that part simpler? I don't know if is important to do these 3 operations in parallel